### ISSUE TITLE
[Feature] Make Errors easier to see

### ISSUE THREAD

**Issue by FDinolfo** [ZDoom/gzdoom#2476]
### GZDoom version

4.12 DEV 471

### Which game are you running with GZDoom?

None

### What Operating System are you using?

Windows 11

### Is your feature request related to a problem? Please describe.

Trying to write ZScript and debug it is a lot harder than it really needs to be. Mostly this is because when I get errors, it is difficult to see them. 

### Describe the solution you'd like

Change the coloring to make them easier to see. I'll attach a screenshot to show what I am seeing.

### Add any other context or screenshots about the feature request here.

Here is a clip of part of the error window to illustrate what I see.

![Screenshot 2024-03-28 122203](https://github.com/ZDoom/gzdoom/assets/6538286/e416cd72-b5cf-4f0b-be12-501fe234ee16)

**Comment by the-phinet** [comment]
By "difficult to see them", do you mean that the contrast is too low?

**Comment by FDinolfo** [comment]
Yes, exactly. A lot of the text on the error window doesn't stand out much.
Warnings and errors specifically.

**Comment by the-phinet** [comment]
Okay, thanks. This will be addressed soon™

**Comment by FDinolfo** [comment]
Much appreciated!
